content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-229.md

```diff
@@ -0,0 +1,108 @@
+---
+weight: 49
+title: 10.18.0.229
+layout: redirect
+---
+
+{{< c8y-admon-info >}}
+These release notes refer to build versions
+- Core: 10.18.0.229
+- UI: 10.18.0.151
+{{< /c8y-admon-info >}}
+
+### Fixes
+
+<table>
+<colgroup>
+<col style="width: 15%;">
+<col style="width:50%;">
+<col style="width: 10%;">
+<col style="width: 12%;">
+<col style="width: 13%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>The <code>mapWidgetPageSize</code> parameter is now also considered if a group is selected for the "Map" widget.</td>
+<td>MTM-55558</td>
+<td>10.18.0.148</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>The <b>Copy dashboard</b> button is now disabled when the user does not have the respective permissions.</td>
+<td>MTM-53334</td>
+<td>10.18.0.140</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>In the "Data points graph" widget, when realtime is enabled and users double-click on a chart, the time range expands now and the data for the current range is shown.</td>

```

Comment:
```suggestion
<td>In the "Data points graph" widget, when realtime is enabled and users double-click on a chart, the time range now expands and the data for the current range is shown.</td>

```

Reply:
Leave as is due to consistency with other releases